public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+      "setup_twoPlayer": "Prenez le paquet Exilés et partisans dans la boîte de jeu et placez-le à côté de la carte. Retirez les quatre cartes Dominance du paquet. Mélangez le jeu."
+    },
+    "standard": {
+      "name": "Standard",
+      "setupTitle": "Configurer le Deck Standard",
+      "setup": "Prenez le paquet Standard dans la boîte de jeu et placez-le à côté de la carte. Mélangez le jeu.",
+      "setup_twoPlayer": "Prenez le paquet Standard dans la boîte de jeu et placez-le à côté de la carte. Retirez les quatre cartes Dominance du paquet. Mélangez le jeu."
+    }
+  },
+  "expansion": {
+    "exilesDeck": "Deck d'Exilés et de Partisans",
+    "landmarkPack": "Pack de Repères",
+    "marauder": "Extension Maraudeur",
+    "marauderHirelings": "Pack de Mercenaires Maraudeurs",
+    "riverfolk": "Extension Riverains",
+    "riverfolkHirelings": "Pack de Mercenaires Riverains",
+    "root": "Root",
+    "underworld": "Extension du Monde Souterrain",
+    "underworldHirelings": "Pack de Mercenaires du Monde Souterrain",
+    "vagabondPack": "Pack des Vagabonds"
+  },
+  "faction": {
+    "alliance": {
+      "name": "Alliance des Bois",
+      "summaryTitle": "Jouer l'Alliance",
+      "summary": "<p>En tant qu'Alliance des Bois, vous travaillez pour gagner plus de sympathie envers les créatures forestières opprimées. Chaque fois que vous placez un <strong>jeton de sympathie</strong> sur la carte, vous marquez des points. Plus il y a de jetons de sympathie sur la carte, plus vous marquez de points.</p><p>Pour gagner de la sympathie, cependant, vous aurez besoin de <strong>supporters</strong> - des cartes placées sur votre plateau de faction - correspondant aux clairières où vous voulez de la sympathie. Bien que vous puissiez ajouter vos propres cartes à vos partisans, vous pouvez inciter vos ennemis à faire de même en provoquant <strong>Outrage</strong>: chaque fois qu'une autre faction retire de la sympathie ou déplace des guerriers dans une clairière sympathique, elle doit ajouter une de ses cartes. propres cartes à vos partisans.</p><p>N'ayez pas peur de mettre en place des points d'étranglement, car vous êtes des experts de la <strong>guerre de guérilla</strong>: lorsque vous défendez au combat, vous utilisez le dé le plus élevé. Placez votre sympathie dans les clairières où le conflit est probable et forcez votre adversaire à vous affronter !</p><p>Les partisans peuvent également être dirigés vers une <strong>révolte</strong> violente, qui détruit <i>tous</i > des pièces de vos ennemis dans une clairière et y place des guerriers et une nouvelle <strong>base</strong>. Les bases augmentent non seulement votre pioche de cartes, mais vous permettent également de former des <strong>officiers</strong>, qui vous donnent des actions gratuites à chaque tour. Protégez bien vos bases ! Si vous perdez une base, vous perdrez également de nombreux partisans et officiers.</p>",
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.